packages/components/src/disabled/README.md

```diff
@@ -33,19 +33,15 @@ const MyDisabled = () => {
 };
 ```
 
-A component can detect if it has been wrapped in a `<Disabled>` by accessing its [context](https://reactjs.org/docs/context.html) using `Disabled.Consumer`.
+A component can detect if it has been wrapped in a `<Disabled>` by accessing its [context](https://reactjs.org/docs/context.html) using `Disabled.Context` and `useContext`.
```

Comment:
I'd personally prefer keeping this sentence a bit more generic (the `useContext` hook is already employed in the code snippet).

```suggestion
A component can detect if it has been wrapped in a `<Disabled />` by accessing its [context](https://reactjs.org/docs/context.html) using `Disabled.Context`.
```

Reply:
Good point 👍